FinalEngine.Editor.ViewModels/Components/PropertyStringViewModel.cs

```diff
@@ -0,0 +1,76 @@
+// <copyright file="PropertyStringViewModel.cs" company="Software Antics">
+// Copyright (c) Software Antics. All rights reserved.
+// </copyright>
+
+namespace FinalEngine.Editor.ViewModels.Components;
+
+using System;
+using CommunityToolkit.Mvvm.ComponentModel;
+
+/// <summary>
+/// Provides a standard implementation of an <see cref="IPropertyStringViewModel"/>.
+/// </summary>
+/// <seealso cref="ObservableValidator" />
+/// <seealso cref="IPropertyStringViewModel" />
+public sealed class PropertyStringViewModel : ObservableValidator, IPropertyStringViewModel
```

Comment:
Instead of creating separating view models for each data type, we can likely get away with creating a generic `EditablePropertyViewModel<T>`.